resources/external_fakes.yaml

```diff
@@ -0,0 +1,20 @@
+description: Butler Data Repository Export
+version: 1.0.1
+data:
+- type: dataset_type
+  name: fakes_fakeSourceCat
+  dimensions:
+  - skymap
+  - tract
+  storage_class: DataFrame
+  is_calibration: false
+- type: dataset
+  dataset_type: fakes_fakeSourceCat
+  run: HSC/external
+  records:
+  - dataset_id:
+    - 9999
```

Comment:
I confess I have no idea what these external yaml files do. I'm guessing 9999 here is just so there's no conflict with other `dataset_id`s?

Reply:
ci_hsc_gen3 now uses UUID as the dataset IDs so 9999 looks very different to the normal expectation.

Reply:
I picked a random number. The external yaml files allow you to specify files that are needed but not created in previous tasks.

Reply:
I can use anything else instead if that will be better?

Reply:
If this is going to be imported with butler import I'm pretty certain that it doesn't matter what the number is in the file so long as within the file it's self-consistent. We'll rewrite on import. You can test that by doing a `butler query-datasets` and looking at the ID after import.